cmd/collaborators/update.go

```diff
@@ -117,3 +107,29 @@ func runUpdateCommand(cmd *cobra.Command, clients *shared.ClientFactory, args []
 
 	return nil
 }
+
+// promptCollaboratorPermissionSelection fetches collaborator permission from the prompt
+func promptCollaboratorPermissionSelection(
+	ctx context.Context,
+	clients *shared.ClientFactory,
+) (
+	permission types.AppCollaboratorPermission,
+	err error,
+) {
+	permissionLabels := []string{
+		"owner",
+		"reader",
+	}
+	response, err := clients.IO.SelectPrompt(
+		ctx,
+		"Select a permission type",
+		permissionLabels,
+		iostreams.SelectPromptConfig{
+			Required: true,
+		},
+	)
```

Comment:
suggestion: I think this needs to be more robust. The `iostreams.SelectPromptConfig` should be bound to the flag and the flag, so that non-interactive environments can work properly with the prompt.